### DISCUSSION TITLE
gocd contribs representin'

### DISCUSSION THREAD

**Discussion by pracplayopen** [gocd/gocd#13367] [General]
SCREENSHOT:  ![image](https://github.com/user-attachments/assets/1bfdae2a-9d90-427a-90bc-fff28d25f8a1)

FULL LIST (100+?): https://github.com/gocd/gocd/graphs/contributors

**Comment by chadlwilson** [comment]
Yes, but sadly not a fair representation of "value" of contribution - a lot of mine are just merge commits of dependabot PRs and small refactorings to keep build/release infra ticking along, as opposed to the chunkier feature development and commit squashing of PRs when GoCD had a dedicated team.

Maybe should change to rebase strategy for PRs :-)
